draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -171,7 +171,7 @@ Both the provider and the service being authenticated and authorized should be u
 
 DNSSEC {{RFC9364}} SHOULD be employed by the domain owner to protect their domain control validation records against DNS spoofing attacks that would let a different (unauthorized) entity successfully complete the validation challenge described by this document.
 
-DNSSEC validation MUST be enabled by service providers that verify domain control validation records they have issued.  When no DNSSEC support is detected for the domain owner zone, service providers verifying records SHOULD attempt to query and confirm by matching the validation record using multiple DNS validators on unpredictable geographically diverse IP addresses to reduce an attacker's ability to complete a challenge by spoofing DNS. Alternatively, service providers MAY perform multiple queries spread out over a longer time period to reduce the chance of receiving spoofed DNS answers.
+DNSSEC validation SHOULD be enabled by service providers that verify domain control validation records they have issued.  When no DNSSEC support is detected for the domain owner zone, service providers verifying records SHOULD attempt to query and confirm by matching the validation record using multiple DNS validators on unpredictable geographically diverse IP addresses to reduce an attacker's ability to complete a challenge by spoofing DNS. Alternatively, service providers MAY perform multiple queries spread out over a longer time period to reduce the chance of receiving spoofed DNS answers.
```

Comment:
I would suggest:

"matching the validation record using multiple DNS validators" to "matching the validation record using multiple  DNS resolvers".

Validators implies doing DNSSEC validation, whereas the multi-vantage point fallback is when we can't do DNSSEC validation.

Also, since we are making the MUST  a SHOULD, we also need explicit text that says that the multi-vantage point confirmation is not only for when there is no DNSSEC support at the domain owner's zone, but also when the service provider (for whatever reason) cannot perform DNSSEC validation.

Thanks!